apps/jobbrowser/src/jobbrowser/apis/query_api.py

```diff
@@ -90,7 +92,8 @@ def apps(self, filters):
             job['duration'],
             re.MULTILINE
         ).groups()),
-        'submitted': job['start_time'],
+        'submitted': datetime.strptime(job['start_time'][:-3], "%Y-%m-%d %H:%M:%S.%f").
```

Comment:
Nit: Could be good to add a comment above how we are changing the start_time here.

Reply:
Sure, added comments. Resolving the conversation.